examples/reference/nuxt.config.js

```diff
@@ -144,7 +144,7 @@ export default {
       username: process.env.CONTENT_API_AUTH_USER,
       password: process.env.CONTENT_API_AUTH_PASS
     },
-    site: 8888,
+    site: 4,
```

Comment:
Added this for testing purpose, as the BE testing branch is in content-vic and this needs to be tested against that. Content-vic has more existing data related to the field change, that is why the test branch is in content-vic BE instead of reference site BE.
**Please revert it back before merging**